### DISCUSSION TITLE
Where did my installed game go?

### DISCUSSION THREAD

**Discussion by hetzjagd** [aldostools/IRISMAN#96] [Q&A] [answered]
First time user.  I thought I had Irisman set to install to /dev_hdd0/GAMES/ but after installing a .pkg through the file manager and it going through it's install process I cannot seem to find where it's installed to, assuming it did actually install.  I have browsed through a few of the folders now and most of them are empty.  The game doesn't show up in the list of installed games within Irisman (I currently have none).  What should I do?

**Comment by aldostools** [comment, marked as answer]
When you install a PKG in IRISMAN, it copies the file to the internal HDD and creates an installation bubble for the PKG that is placed in Games column > Package Manager >  PlayStation®Network Content.

You should install the PKG using the bubble on XMB.

Once the PKG is installed, the PSN game is stored in /dev_hdd0/game/`NPXX12345` <= replace this with the correct title id of the game

**Reply by gooddavvy** [reply]
What is this?
